services/src/apidoc.rs

```diff
@@ -128,6 +128,8 @@ use utoipa::{Modify, OpenApi};
             DataId,
             DataProviderId,
             DatasetId,
+            DatasetName,
+            NamedData,
```

Comment:
You need to adjust the pro apidoc.rs too

Reply:
Didn't I do this? Isn't there a CI check for this?

Reply:
no you didn't. At least I cannot find `NamedData` in src/pro/apidoc.rs